actions-sdk-app.js

```diff
@@ -111,12 +111,12 @@ class ActionsSdkApp extends AssistantApp {
         error(errorMsg);
         reject(errorMsg);
       }
-      googleAuthClient.verifyIdToken(jwtToken, projectId, (err, idToken) => {
+      googleAuthClient.verifyIdToken(jwtToken, projectId, (err, login) => {
         if (err) {
           error('ID token verification Failed: ' + err);
           reject(err);
         } else {
-          resolve(idToken);
+          resolve(jwtToken);
```

Comment:
This seems to be a breaking change, can you explain the reason for returning the jwtToken?

Reply:
The issue here is the implementation didn't match the documentation. From the documentation:
> Promise resolving with ID token if request is from a valid source

Here's what the call in question looked like:
```
  googleAuthClient.verifyIdToken(jwtToken, projectId, (err, idToken) => {
```

Here's what the actual signature of that method looks like:
```
  public verifyIdToken(
    idToken: string, audience: string\|string[],
    callback: (err: Error, login?: LoginTicket) => void) {
```

So there are two problems here.
1. The second callback parameter is a `LoginTicket` not an ID token. Returning that conflicts with the documentation.
1. If you want to return what googleAuthClient considers to be the ID token, then it's the first parameter, which in this case is `jwtToken`.